src/main/kotlin/no/nav/syfo/service/microfrontend/MikrofrontendDialogmoteService.kt

```diff
@@ -0,0 +1,156 @@
+package no.nav.syfo.service.microfrontend
+
+import no.nav.syfo.db.*
+import no.nav.syfo.db.domain.toMikrofrontendSynlighet
+import no.nav.syfo.kafka.consumers.varselbus.domain.ArbeidstakerHendelse
+import no.nav.syfo.kafka.consumers.varselbus.domain.HendelseType
+import no.nav.syfo.kafka.consumers.varselbus.domain.getSynligTom
+import no.nav.syfo.kafka.producers.mineside_microfrontend.MikrofrontendSynlighet
+import no.nav.syfo.kafka.producers.mineside_microfrontend.MinSideRecord
+import no.nav.syfo.kafka.producers.mineside_microfrontend.Tjeneste
+import no.nav.syfo.service.microfrontend.MikrofrontendService.Companion.actionDisabled
+import no.nav.syfo.service.microfrontend.MikrofrontendService.Companion.actionEnabled
+import no.nav.syfo.utils.DuplicateMotebehovException
+import no.nav.syfo.utils.MotebehovAfterBookingException
+import no.nav.syfo.utils.VeilederAlreadyBookedMeetingException
+import org.slf4j.LoggerFactory
+
+class MikrofrontendDialogmoteService(
+    val database: DatabaseInterface
+) {
+    private val log = LoggerFactory.getLogger(MikrofrontendService::class.java)
+
+    companion object {
+        const val dialogmoteMikrofrontendId = "syfo-dialog"
+    }
+
+    fun updateDialogmoteFrontendForUserByHendelse(hendelse: ArbeidstakerHendelse, ferdigstill: Boolean): MinSideRecord? {
+        if (hendelse.type == HendelseType.SM_DIALOGMOTE_NYTT_TID_STED) {
```

Comment:
Forslag: Erstatte if med when/else

Reply:
I  Kotlin code conventions anbefales det å [foretrekke ](https://kotlinlang.org/docs/coding-conventions.html#nullable-boolean-values-in-conditions) `when` når det er 3 eller flere options

Reply:
Notert 👍

Reply:
@AudunSorheim Fikser 👍